README.md

```diff
@@ -33,8 +33,71 @@
 
 ### Dependencies
 
-The **Make** targets assume you have **bash**, **curl**, **tar**, **gopass**, **summon**, **gopass summon provider**, **docker** and **docker-compose** installed.
+<!--The **Make** targets assume you have **bash**, **curl**, **tar**, **gopass**, **summon**, **gopass summon provider**, **docker** and **docker-compose** installed.-->
+Prerequisites for development:
+- a recent python version (>= 3.7) (check with `python -V`)
```

Comment:
Don't match the version in makefile which is 3.6

Reply:
fixed